client-core/src/wallet/syncer.rs

```diff
@@ -1337,50 +1342,115 @@ mod tests {
     }
 }
 
+// nodelist_info : example)  "nodeid@ip:port,nodeid@ip:port"
+fn parse_node(nodelist_info: &str) -> Result<Vec<(String, String, u32)>> {
+    let split_nodes = nodelist_info.split(',');
+    let mut all_nodes: Vec<(String, String, u32)> = vec![];
+    for a_node in split_nodes.into_iter() {
+        let split_nodeid: Vec<String> = a_node.split('@').map(|a| a.to_string()).collect();
+        if 2 != split_nodeid.len() {
+            return Err(Error::new(
+                ErrorKind::InvalidInput,
+                format!("invalid nodelist_info nodeid {}", a_node),
+            ));
+        }
+        let nodeid = split_nodeid[0].clone();
+        let ip_port = split_nodeid[1].clone();
+        ip_port.split(':');
+        let split_ip: Vec<String> = ip_port.split(':').map(|a| a.to_string()).collect();
+        if 2 != split_ip.len() {
+            return Err(Error::new(
+                ErrorKind::InvalidInput,
+                format!("invalid nodelist_info ip_port {}", ip_port),
+            ));
+        }
+        let ip = split_ip[0].clone();
+        let port = split_ip[1]
+            .parse::<u32>()
+            .chain(|| (ErrorKind::InvalidInput, "Unable to parse node string"))?;
+        all_nodes.push((nodeid, ip, port));
+    }
+    Ok(all_nodes)
+}
+
 /// [new light client design](https://github.com/informalsystems/tendermint-rs/blob/master/docs/architecture/adr-006-light-client-refactor.md)
 pub fn spawn_light_client_supervisor(
     db_path: &Path,
-    addr: &str,
+    // ws://localhost:26657/websocket
+    _websocket_addr: &str,
```

Comment:
this argument is not necessary?

Reply:
now it's not necessary, because , we don't use `default` values,
i'll remove it